src/core/packages/rendering/server-internal/src/rendering_service.tsx

```diff
@@ -60,8 +61,14 @@ type RenderOptions =
 
 const themeVersion: ThemeVersion = 'v8';
 
+// TODO: Remove the temporary feature flag and supporting code when Borealis is live in Serverless
+// https://github.com/elastic/eui-private/issues/192
+export const DEFAULT_THEME_NAME_FEATURE_FLAG = 'coreRendering.defaultThemeName';
+
 /** @internal */
 export class RenderingService {
+  private readonly themeName$ = new BehaviorSubject<ThemeName>(DEFAULT_THEME_NAME);
```

Comment:
nit: I wonder if this should be a `ReplaySubject(1)` without a pre-defined value instead.

This way, it should hold any requests resolving the theme until `getStringValue$` is executed (to avoid flaky behaviors).

```suggestion
  private readonly themeName$ = new ReplaySubject<ThemeName>(1);
```

WDYT?

Reply:
Talked offline... this might break the preboot renderer. Let's keep the initial value for now.